src/RichTextEditor/FormattingToolbar/HyperlinkModal.js

```diff
@@ -114,6 +116,11 @@ const HyperlinkModal = React.forwardRef(({ ...props }, ref) => {
     props.setShowLinkModal(false);
   };
 
+  const handleUrlInput = (event) => {
+    Transforms.select(editor, originalSelection);
+    event.target.value ? setApplyStatus(true) : setApplyStatus(false);
```

Comment:
here you can use the same syntax to simplify by making this line just `setApplyStatus(!!event.target.value)`